content/pages/publishAlgo.json

```diff
@@ -0,0 +1,69 @@
+{
+  "title": "Publish",
+  "description": "Highlight the important features of your algorith to make it more discoverable and catch the interest of data consumers.",
+  "warning": "Given the beta status, publishing on Ropsten or Rinkeby first is strongly recommended. Please familiarize yourself with [the market](https://oceanprotocol.com/technology/marketplaces), [the risks](https://blog.oceanprotocol.com/on-staking-on-data-in-ocean-market-3d8e09eb0a13), and the [Terms of Use](/terms).",
+  "form": {
+    "title": "Publish",
```

Comment:
I think we do not output that form title right now but with 2 forms to switch between, we should do that. So that the form title switches between "Publish a data set" & "Publish an algorithm". And we might want to have a form description under that title too so we could already prepare that. Cause then we can keep the page description more generic, and use those form's title & description to explain the difference between them individually